examples/getting-started-movielens/02-ETL-with-NVTabular.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    workflow.save(os.path.join(INPUT_DATA_DIR, "workflow"))
</pre><p>Not a big deal, but if you want, you can also add output schema here as well to print out the schema in the cell. like <code>workflow.output_schema</code>  should do the job.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
added the line to shout the output schema